storybookjs/storybook
# removed yarn workspaces and added packages to lerna.json

Issue: https://github.com/storybooks/storybook/issues/4204

## What I did
I removed the yarn workspaces config from package.json and added the directory paths to lerna.json

## How to test
I have manually tested the `publish:alpha` npm script, which finished without error. 

Is this testable with Jest or Chromatic screenshots? No
Does this need a new example in the kitchen sink apps? No
Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
